doc: Add Mikrotik KNOT documentation (#321)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/mikrotik-knot/_index.md

---
title: "MikroTik KNOT"
description: ""
weight: 1
---

> {{< figure src="mikrotik-knot.jpeg" alt="Ltmikrotik-knotAP" class="plain float" >}}

Reviewer: You should move this image, as it gets covered by the warning about changing frequency plans

Reviewer 2: <img width="1391" alt="Bildschirmfoto 2021-05-03 um 13 50 09" src="https://user-images.githubusercontent.com/6963436/116868407-68f45380-ac17-11eb-9ac3-012c66b2e83e.png">

> The MikroTik KNOT LR8/LR9 kit is a universal device with exceptional connectivity options and protocol support. It is an IoT Gateway which can use LTE-M (also known as CAT-M) technology as the gateway backhaul. 

Reviewer: What about other technologies for backhaul?

<!--more-->

> For more info, see Mikrotik's [official product page](https://mikrotik.com/product/knot)

Reviewer: ```suggestion
For more info, see Mikrotik's [official product page](https://mikrotik.com/product/knot).
```

{{< warning >}} Currently, this gateway does not support changing Frequency Plans. {{</ warning >}}

## Prerequisites

1. User account on {{% tts %}} with rights to create Gateways.

> 2. MikroTik KNOT connected via Ethernet or LTE-M.

Reviewer: ```suggestion
2. MikroTik KNOT connected to Internet via Ethernet or LTE-M.
```

3. A web browser (see section [Configuration via Browser](#configuration-via-browser)), or the [MikroTik Mobile App](https://mikrotik.com/mobile_app) installed on a smartphone.

## Registration

Create a gateway by following the instructions for [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}). The **EUI** of the gateway can be found on the back panel of the gateway under the field **GW ID**, or under the **Gateway ID** in the LoRa tab of the MikroTik web GUI.

## Configuration via Browser

> The MikroTik Routerboard exposes a WiFi Access Point (AP) with SSID `MikroTik-xxxxxx`, where `xxxxxx` are the last 6 digits of the device's MAC Address.
> 
> Use your PC, tablet or smartphone to connect to this AP. You will not need a password to connect to this AP.

Reviewer: ```suggestion
The MikroTik Routerboard exposes a WiFi Access Point (AP) with SSID `MikroTik-xxxxxx`, where `xxxxxx` are the last 6 digits of the device's MAC address. Use your PC, tablet or smartphone to connect to this AP. You will not need a password to connect to this AP. 

{{< info >}} By opening your browser to `http://192.168.88.1/webfig` you will be able to access the RouterOS web interface and configure your gateway device. {{</ info }}
```

Open a browser to `http://192.168.88.1/webfig/#Interfaces`.

![MikroTik-portal.png](portal.png)

Select **LoRa** on the left hand menu. The status of the gateway should be **Disabled** - if needed, press the **D** button to disable it.

![MikroTik-portal-lora.png](portal-lora.png)

{{< note >}}

> The Gateway ID you see here is the Gateway EUI you need to register the gateway in The Things Stack.

Reviewer: ```suggestion
The **Gateway ID** you see here is the **Gateway EUI** you need to register the gateway in The Things Stack.
```

{{</ note >}}

> From the top menu, select the **Servers** Tab and press the **Add New** button.

Reviewer: ```suggestion
From the top menu, select the **Servers** tab and press the **Add New** button.
```

![MikroTik-server](servers.png)

Edit the server parameters.

> - **Name**: A distinct name

Reviewer: Is this something that the user defines arbitrarily?

> - **Address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.

Reviewer: ```suggestion
- **Address**: Address of {{% tts %}} Gateway Server. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
```

Reviewer: What does one use if he didn't follow the getting started guide? For example if he's using TTS Community Edition?

- **Up port**: UDP upstream port of the Gateway Server, typically `1700`.
- **Down port**: UDP downstream port of the Gateway Server, typically `1700`.

Press **OK** to save the parameters.

![MikroTik-paramters](parameters.png)

> From the top menu, select the **Devices** tab. Click the **Gateway ID** to configure it.

Reviewer: To configure the gateway ID or the device itself?

![MikroTik-Devices](portal-lora-disable.png)

> In the device settings, click the **Network Servers** drop-down menu to view available servers. Choose the server you just created.

Reviewer: ```suggestion
In the device settings, click the **Network Servers** drop-down menu to view available servers. Choose the server you created in the previous step.
```

Select **Public** for the **Network** and click **OK** to save these parameters.

Press the **E** button to enable the gateway device (it is enabled when **E** becomes **D**) and the device status will update.

If your configuration was successful, your gateway will connect to {{% tts %}} after a couple of seconds.

> ## Set up LET-M

Reviewer 2: ```suggestion
## Set up LTE-M
```

Connect an external antenna to the LTE-M antenna connector.

> <img src="KNOT-antenna.png" alt="KNOT Antennas"	title="KNOT Antennas" width="250" />

Reviewer: This image could be floating, I think it would look nicer

Author: I tried, didn't work well

> 

Reviewer: ```suggestion
```

> Go to **Interfaces**, click on **PPP-out1** and fill out the **APN settings** of your operator.

Reviewer: Does "Go to Interfaces" mean the same as "Open a browser to http://192.168.88.1/webfig/#Interfaces" in the text above? If yes, I prefer you use the second one, as it is more clear

> To test whether a connection is made to your LTE-M network, go to the Terminal in the RouterOS and run the command:

Reviewer: ```suggestion
To test whether a connection with your LTE-M network is established, go to the **Terminal** tab in the RouterOS web interface and run the command:
```

```
$ /interface ppp-client info ppp-out1
```

> If the gateway is set up correct, the gateway will respond with a message like:

Reviewer: ```suggestion

If the gateway is set up correctly, it will respond with a message like:
```
